### ISSUE TITLE
user supplied schema ChainMapped to default schema?

### ISSUE THREAD

**Issue by sbillinge** [regro/regolith#273]
@scopatz @CJ-Wright  is there a reason that in regolith validate the default schema is only updated by the user-supplied one and not chainmapped?
```
    if "schemas" in rc._dict:
        user_schema = copy.deepcopy(rc.schemas)
        rc.schemas = copy.deepcopy(SCHEMAS)
        for k in user_schema:
            for k2, v in user_schema[k].items():
                rc.schemas[k][k2].update(v)
```
We have a UC where we want to add keys to the schema locally but then validate against this updated schema and we can't do it with the current approach.

**Comment by CJ-Wright** [comment]
Not to my knowledge, PRs welcome!

**Comment by sbillinge** [comment]
I have Songsheng working on a PR......I just wanted to check that it wasn't
a design decision before we put in too much effort.

**Comment by sbillinge** [comment]
closed by #278
